eng/restore/optimizationData.targets

```diff
@@ -34,4 +34,50 @@
 
   </Target>
 
+
+  <ItemGroup>
+    <!-- Mibc data to use when exact architecture match is available -->
+    <MIBCPackageDef Include="optimization.windows_nt-x86.MIBC.Runtime" Version="$(optimizationwindows_ntx64MIBCRuntimeVersion)" MibcArchitecture="Windows/x86"/>
```

Comment:
Should "optimizationwindows_ntx64MIBCRuntimeVersion" be "optimizationwindows_ntx86MIBCRuntimeVersion" instead in line 40?

Reply:
Yes. Fixed.